dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05305AddPushPublishFilterColumn.java

```diff
@@ -0,0 +1,42 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotmarketing.startup.AbstractJDBCStartupTask;
+import java.util.List;
+
+public class Task05305AddPushPublishFilterColumn extends AbstractJDBCStartupTask {
+
+    @Override
+    public String getPostgresScript() {
+        return "ALTER TABLE publishing_bundle ADD filter_key VARCHAR(100)";
+    }
+
+    @Override
+    public String getMySQLScript() {
+        return "ALTER TABLE publishing_bundle ADD filter_key VARCHAR(100)";
+    }
+
+    @Override
+    public String getOracleScript() {
+        return "ALTER TABLE publishing_bundle ADD filter_key VARCHAR2(100)";
+    }
+
+    @Override
+    public String getMSSQLScript() {
+        return "ALTER TABLE publishing_bundle ADD filter_key NVARCHAR(100)";
+    }
+
+    @Override
+    public String getH2Script() {
+        return null;
+    }
+
+    @Override
+    protected List<String> getTablesToDropConstraints() {
+        return null;
+    }
+
+    @Override
+    public boolean forceRun() {
+        return true;
```

Comment:
I would ask to the catalog if the filter_key does not exists on publishing_bundle to proceed